plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/META-INF/MANIFEST.MF

```diff
@@ -7,13 +7,15 @@ Bundle-SymbolicName: com.google.cloud.tools.eclipse.appengine.newproject.test
 Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.newproject
-Require-Bundle: org.junit;bundle-version="4.12.0",
+Require-Bundle: com.google.cloud.tools.eclipse.appengine.flex,
```

Comment:
This is the same problem with https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1660#discussion_r108299807. This test attempts to install the flex facet, which requires the flex facet definition. However, the `.flex` bundle isn't in our target platform yet; this directive will pull in the bundle in testing.

When we eventually remove the `.flex` bundle, Eclipse will place an error marker in this file, so we won't forget removing this later.